dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -54,3 +57,13 @@
                     %span.puzzle-number
                       = @script_level.position
                 %td.name= link_to student.name, build_script_level_path(@script_level, user_id: student.id, section_id: @section.id)
+  :javascript
+    function resizeScrollable() {
+      var newHeight = $('.teacher-panel').height() -
```

Comment:
I think the magic pixels are coming from margins of things that are lower in the hierarchy than what I'm using... what I want logically is innerHeight of div.content as if div.content's size was determined by its container rather than its contents.